docs/architecture/modernize-desktop/whats-new-dotnet-core-desktop.md

```diff
@@ -0,0 +1,149 @@
+
+The good news is Microsoft plans to support both .NET Framework and .NET Core. In fact, the announcement of .NET Core 3.0 was followed by exciting features in .NET Framework regarding modernization, so let's take a brief look at both paths.
+
+Listening to the .NET Framework developer's community suggestions, Microsoft have addressed three main scenarios for **.NET Core**:
+
+- **Side-by-side versions of .NET supporting Windows Forms and WPF: This solves the problem of side effects when updating the machine's framework version. With .NET Core multiple versions can be installed on the same machine and for each application you can specify, which version of .NET Core you'd like it to use. Even more, now you can develop and run Windows Forms and WPF on top of .NET Core.
+- **Embed .NET directly into an application: You can now ship the .NET Core as part of your application package. This enables you to take advantage of the latest version, features, and APIs without having to wait for a specific version to be installed on the machine.
+- **Take advantage of .NET Core features: .NET Core is the fast-moving, open-source version of .NET. Its side-by-side nature enables fast introduction of new innovative APIs and BCL (Base Class Libraries) improvements without the risk of breaking compatibility. Now Windows Forms and WPF applications on Windows can take advantage of the latest .NET Core features, which also includes more fundamental fixes for runtime performance, high-DPI support, and so on.
+
+Regarding **.NET Framework 4.8**, Microsoft has addressed three scenarios:
+
+- **Modern browser and media controls: Today, .NET desktop applications use Internet Explorer and Windows Media Player for showing HTML and playing media files. Since these legacy controls don't show the latest HTML or play the latest media files, we are adding new controls that take advantage of Microsoft Edge and newer media players to support the latest standards.
+- **Access to UWP Controls: UWP contains new controls that take advantage of the latest Windows features and touch displays. You will not have to rewrite your applications to use these new features and controls. Now they are available to Windows Forms and WPF so that you can take advantage of these new features in your existing code.
+- **High DPI improvements: The resolution of displays is increasing to 4K and 8K resolutions. We want to make sure your existing Windows Forms and WPF applications can look great on these new displays. That's why we are adding improvements for HDPI
+
+Since .NET Framework is working on millions of machines, Microsoft will continue to support it but will not be adding new features.
+
+.NET Core is the open source, cross-platform, and fast-moving version of .NET. Because of its side-by-side nature, it can take changes without the fear of breaking any application. This means that .NET Core will get new APIs and language features over time that .NET Framework cannot.
```

Comment:
```suggestion
.NET Core is the open-source, cross-platform, and fast-moving version of .NET. Because of its side-by-side nature, it can take changes without the fear of breaking any application. This means that .NET Core will get new APIs and language features over time that .NET Framework won't.
```